### PR TITLE
Assignment 1 API.

### PR DESCRIPTION
**Pull request by OkayNiceBrian**
Assignment1 part 2.

image

### CODE REVIEW

**Comment by TechGlue** [comment]
I will review this

**Comment by TechGlue** [comment]
Hi, Brian!

API project updated to output new welcome message in second pull request: ✔
Screenshot of web browser running the API project included in second pull request: ✔

Changes to the API project made and the output of the changes are shown. Great work! - Luis

**Review by TechGlue — COMMENTED** [review]
Hi, Brian!

API project updated to output new welcome message in second pull request: ✔
Screenshot of web browser running the API project included in second pull request: ✔

Changes to the API project made and the output of the changes are shown. Great work! - Luis

File: SecretSanta/src/SecretSanta.Api/Startup.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapGet("/", async context =>
                 {
-                    await context.Response.WriteAsync("Hello from API!");
+                    await context.Response.WriteAsync("Hello from your pal Brian! (API)");
```
Comment (mmwoodfo) [inline SecretSanta/src/SecretSanta.Api/Startup.cs:35]: :ok_hand:

**Pull request closed without merging**